registry.go

```diff
@@ -11,6 +11,25 @@ type RegistryClient struct {
 	*Client
 }
 
+// CreateRegistry
+// Create a registry
+func (s *RegistryClient) CreateRegistry(r Registry) error {
+	resp, _, errs := s.NewRequest(gorequest.POST, "").
+		Send(r).
+		End()
+	return CheckResponse(errs, resp, 201)
+}
+
+// ListRegistries
+// Get a list of registries
+func (s *RegistryClient) ListRegistries(name string) ([]Registry, error) {
+	var r []Registry
+	resp, _, errs := s.NewRequest(gorequest.GET, "").
+		Query(name).
```

Comment:
What does the `name` parameter actually do? If used in a query string, shouldn't this be `Query(map[string]string{"name": name})` or something similar?

Reply:
Yeah, it does query the API, though the suggested way is more elegant.
Found some other places where this was not happening, too.